package.json

```diff
@@ -221,6 +221,7 @@
     "docker-toolbelt": "^3.3.8",
     "dockerode": "^2.5.8",
     "ejs": "^3.1.3",
+    "electron-notarize": "^1.0.0",
```

Comment:
This was added to the `dependencies` section of `package.json`, but I think it should be added to the `devDependencies` section instead (because it's not needed when CLI end users run `npm i -g --production balena-cli`).